Mac Apple M1 chip Support

https://qiita.com/ucan-lab/items/38cd04cee1f3f9e024b9#comment-9226685e6778df2e729a

```
$ docker-compose exec web php artisan migrate:fresh --seed

   Illuminate\Database\QueryException 

  SQLSTATE[HY000] [2002] Connection refused (SQL: SHOW FULL TABLES WHERE table_type = 'BASE TABLE')

  at vendor/laravel/framework/src/Illuminate/Database/Connection.php:678
    674▕         // If an exception occurs when attempting to run a query, we'll format the error
    675▕         // message to include the bindings with SQL, which will make this exception a
    676▕         // lot more helpful to the developer instead of just the database's errors.
    677▕         catch (Exception $e) {
  ➜ 678▕             throw new QueryException(
    679▕                 $query, $this->prepareBindings($bindings), $e
    680▕             );
    681▕         }
    682▕ 

      +44 vendor frames 
  45  artisan:37
      Illuminate\Foundation\Console\Kernel::handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
make[2]: *** [fresh] Error 1
make[1]: *** [init] Error 2
make: *** [remake] Error 2
```

## Change MySQL Docker Image

`mysql` to `mysql/mysql-server`

- https://hub.docker.com/_/mysql
- https://hub.docker.com/r/mysql/mysql-server

--- a/infra/docker/mysql/Dockerfile
+++ b/infra/docker/mysql/Dockerfile
@@ -1,4 +1,4 @@
-FROM mysql:8.0
+FROM mysql/mysql-server:8.0
 LABEL maintainer="ucan-lab <yes@u-can.pro>"
 
 ENV TZ=UTC

--- a/infra/docker/mysql/my.cnf
+++ b/infra/docker/mysql/my.cnf
@@ -1,4 +1,5 @@
 [mysqld]
+user=mysql
 character_set_server = utf8mb4
 collation_server = utf8mb4_0900_ai_ci
 

